test/parallel/test-stream-readable-needReadable.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const Readable = require('stream').Readable;
+
+const readable = new Readable({
+  read: () => {}
+});
+
+// Initialized to false.
+assert.strictEqual(readable._readableState.needReadable, false);
+
+readable.on('readable', common.mustCall(() => {
+  // When the readable event fires, needReadable is reset.
+  assert.strictEqual(readable._readableState.needReadable, false);
+  readable.read();
+}), 1);
```

Comment:
Thank you for pointing this out. That's meant for `common.mustCall` but I accidentally put it inside the wrong parentheses. The default is 1 anyway so I agree with you, this can be removed.